js/huobi.js

```diff
@@ -2845,6 +2872,33 @@ module.exports = class huobi extends Exchange {
         return result;
     }
 
+    networkIdToCode (networkId, currencyCode = undefined) {
+        // here network-id is provided as a pair of currency & chain (i.e. trc20usdt)
+        if (currencyCode === undefined) {
+            throw new ArgumentsRequired (this.id + ' networkIdToCode() requires a currencyCode argument');
+        }
+        const keysLength = (Object.keys (this.networkTitlesByJunctions)).length;
+        if (keysLength === 0) {
```

Comment:
if we need the markets loaded, why not add `await loadMarkets()` at the beginning of the method instead of throwing?

Reply:
for two reasons:
1) the method is synchoronous and not meant to be async
2) that method is being used inside `loadMarkets -> fetchCurrencies` itself, so it will make just recursion if we call loadMarkets from there.

Reply:
@ttodua ok, in that case I think it's reasonable to throw